src/assets/data/markdown/ionic-weather-step02.md

```diff
@@ -205,13 +207,13 @@ The iOS build may fail in the packaging step, but that is OK.
 
 **Note:** you will need to put your device in "Developer Mode" if it is not already
 
-1. conntect your device
+1. connect your device
 1. `adb devices` - make sure your device is listed
 1. `ioinc cordova run android` - this will build your application and run it on your attached Android device
 
 **Note:** it is also possible to use Android Studio to build and run the application
 
-## Squash and Merge
+## Squash (Optional) and Merge (Required)
```

Comment:
Totally understand where you're coming from here - I love having a clean commit history as much as the next dev - but is this overkill for the training? 🤔 

I added Optional & Required to denote that folks don't have to do the Squash but it's a nice-to-have... thoughts?

Reply:
I agree. I was actually going to point out that I don't care what they do with `git` during training, but to suggest it is always a good habit to get into.

I like the idea of having the (Optional) and (Required) in the docs to make it more obvious that one of those is just best practice while the other actually must be done in order for stuff to work....

Reply:
Not a requirement by any means but figured in this particular case it may make sense :)

Reply:
Yeah. In reality, branching and merging are not required for the training at all, so I am going to suggest but not require it. I would like to explain to them though that it is a good idea, even on "home" and "experimental" projects.

I can't count the number of times I started a branch and said "nope, that's crap" and it is so much easier  to just toast the whole branch than to rollback a bunch of commits in `master`. So that's my plan, anyhow.